FirebaseAppDistribution/Sources/FIRAppDistribution.m

```diff
@@ -71,8 +71,8 @@ - (instancetype)initWithApp:(FIRApp *)app appInfo:(NSDictionary *)appInfo {
 
   if (self) {
     [GULAppDelegateSwizzler proxyOriginalDelegate];
-    self.appDelegateInterceptor = [FIRAppDistributionAppDelegateInterceptor sharedInstance];
-    [GULAppDelegateSwizzler registerAppDelegateInterceptor:self.appDelegateInterceptor];
+    self.UIService = [FIRAppDistributionUIService sharedInstance];
```

Comment:
nit: uiService - Jeremy and I looked this up, and it looks like you can either do this or do lower case when an acronym begins a variable. My preference is lower case, though it's not a hill I'm willing to die on.

Reply:
Yeah was not sure about that. Will update.